### PR TITLE
track connectors versions

### LINKED ISSUE #2615: Track the connector version for syncs in Segment
##Tell us about the problem you're trying to solve
When we identify issues with connectors, it could be interesting to know which connector version they are. 

Describe the solution you’d like
Add properties connector_version for Connector Jobs when job_type=SYNC.

Here's the updated Segment spreadsheet: https://docs.google.com/spreadsheets/d/1qQPMBdz86yMBiyXpjw91D3HoP6hN7yIPRmLrF7xWOgA/edit#gid=1567609168

@cgardens

### PR DESCRIPTION
**Pull request by ChristopheDuong**
## What
Closes #2615

### CODE REVIEW

File: airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/JobTracker.java [hunk 1]
```diff
@@ -185,7 +185,10 @@ public void trackSync(Job job, JobState jobState) {
     final StandardDestinationDefinition destinationDefinition = configRepository.getStandardDestinationDefinition(destinationDefinitionId);
     metadata.put("connector_destination", destinationDefinition.getName());
     metadata.put("connector_destination_definition_id", destinationDefinition.getDestinationDefinitionId());
-
+    final String[] imageTag = destinationDefinition.getDockerImageTag().split(":");
```
Comment (ChristopheDuong) [inline airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/JobTracker.java:188]: Should it be the docker image (version) tag only? or the full docker image name instead?

File: airbyte-analytics/src/main/java/io/airbyte/analytics/LoggingTrackingClient.java [hunk 2]
```diff
@@ -57,7 +57,10 @@ public void track(String action) {
 
   @Override
   public void track(String action, Map<String, Object> metadata) {
-    LOGGER.info("track. userId: {} action: {}, metadata: {}", identitySupplier.get().getCustomerId(), action, metadata);
+    LOGGER.info("track. version: {}, userId: {}, action: {}, metadata: {}",
+        identitySupplier.get().getAirbyteVersion(),
+        identitySupplier.get().getCustomerId(),
+        action, metadata);
```
Comment (cgardens) [inline airbyte-analytics/src/main/java/io/airbyte/analytics/LoggingTrackingClient.java:63]:
```suggestion
        action, 
        metadata);
```

File: airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/JobTracker.java (hunk 1 above)
Comment (cgardens) [reply airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/JobTracker.java:188]: i think we should do one of the following. 
1. full docker image name with tag e.g. `connector_destination_version ` = `airbyte/postgres-source:0.2.0` 
2. pull the docker image name and tag but put them in separate fields `connector_destination_image_name` = `airbyte/postgres-source`and `connector_destination_image_version`= `0.2.0`

The one thing I feel strongly about is just saving the tag is enough enough.

**Review by cgardens — APPROVED** [review]

File: airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/JobTracker.java (hunk 1 above)
Comment (ChristopheDuong) [reply airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/JobTracker.java:188]: Seems like John is ok with just the version tag `0.2.0`

**Pull request merged**